src/api/middlewares/AuthenticateMiddleware.ts

```diff
@@ -18,54 +18,70 @@ export class AuthenticateMiddleware implements interfaces.Middleware {
     }
 
     public use = (req: myExpress.Request, res: myExpress.Response, next: myExpress.NextFunction): void => {
-        const token = this.getToken(req);
+        const authentication = auth(req);
 
-        if (token === null) {
-            this.log.warn('No token given');
-            return res.failed(403, 'You are not allowed to request this resource!');
+        if (!authentication || !this.isAuthenticate(authentication)) {
+            this.log.warn('You are not allowed to request this resource!');
+            return res.failed(401, 'You are not allowed to request this resource!');
         }
-        this.log.debug('Token is provided');
+        return next();
+    }
```

Comment:
do this the other way:
if( isAuthenticated() ) return next()
and everything else: fail